src/style.css

```diff
@@ -0,0 +1,85 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.search_input {
+  font-family: inherit;
+  width: 100%;
+  padding: 0;
+  font-weight: 300;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+  color: #3d4e61;
+}
+
+.searchBig {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  font-family: Avenir, sans-serif;
+}
+
+.searchSmall {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
```

Comment:
you can declare styles for big input just searct_input and than re-declare them for small input. So in result you have only .search_input and .searchSmall

Reply:
I'm corrected errors in css file. Please chek my work)